lite2/client.go

```diff
@@ -84,11 +84,12 @@ func SkippingVerification(trustLevel tmmath.Fraction) Option {
 	}
 }
 
-// AlternativeSources option can be used to supply alternative providers, which
-// will be used for cross-checking the primary provider.
-func AlternativeSources(providers []provider.Provider) Option {
+// Witnesses option can be used to supply providers, which will be used for
+// cross-checking the primary provider. A witness can become a primary iff the
```

Comment:
*picky:* A witness can become a primary iff -> A witness can become a primary if

Reply:
iff - if and only if

Reply:
ah I see. Thanks :)